### PR TITLE
Clean up version number and history following the v1.18.6 release

### PR DESCRIPTION
**Pull request by RyanTheOptimist**
Leaves the branch ready for a potential subsequent v1.18.7 release.

Signed-off-by: Ryan Hamilton <rch@google.com>

### CODE REVIEW

**Comment by RyanTheOptimist** [comment]
This is the first of 4 similar PRs I will be sending you. Feel free to only review one of them, if this approach is not correct.

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-1.18.6-dev
+1.18.7
```
[the remark this answers is not in the document]
Comment (RyanTheOptimist) [reply VERSION]: Sure, that works or me. Done!

**Review by mattklein123 — APPROVED** [review]
Thanks!

**Pull request merged**